pipeline_tools/pipelines/optimus/optimus.py

```diff
@@ -55,6 +55,28 @@ def create_optimus_input_tsv(uuid, version, dss_url):
     sample_id = metadata_utils.get_sample_id(primary_bundle)
     print('Writing sample ID to sample_id.txt')
     with open('sample_id.txt', 'w') as f:
-        f.write('{0}'.format(sample_id))
+        f.write(f"{sample_id}")
+
+    species_references = references[metadata_utils.get_ncbi_taxon_id(primary_bundle)]
+    print('Writing species references')
+    for key, value in species_references.items():
+        with open(f"{key}.txt", 'w') as f:
+            f.write(f"{value}")
 
     print('Finished writing files')
+
+
+references = {
+    # human
+    9606: {
+        'tar_star_reference': 'gs://hca-dcp-sc-pipelines-test-data/alignmentReferences/optimusGencodeV27/buildReference/output_bucket/star_primary_gencode_v27.tar',
+        'annotations_gtf': 'gs://hca-dcp-sc-pipelines-test-data/alignmentReferences/optimusGencodeV27/gencode.v27.primary_assembly.annotation.gtf.gz',
+        'ref_genome_fasta': 'gs://hca-dcp-sc-pipelines-test-data/alignmentReferences/optimusGencodeV27/GRCh38.primary_assembly.genome.fa',
+    },
+    # mouse
+    10090: {
+        'tar_star_reference': 'gs://hca-dcp-mint-test-data/20190507-PipelinesSurge/mouse_reference/star_primary_gencode_mouse_vM21.tar',
+        'annotations_gtf': 'gs://hca-dcp-mint-test-data/yanc-test/gencode.vM21.annotation.gtf.gz',
```

Comment:
Is there a reason why you didn't use `gs://hca-dcp-mint-test-data/20190507-PipelinesSurge/mouse_reference/gencode.vM21.annotation.gtf.gz` and `gs://hca-dcp-mint-test-data/20190507-PipelinesSurge/mouse_reference/GRCm38.primary_assembly.genome.fa.gz` here?

If you need the uncompressed `fa` file, could you put it in `mouse_reference` location, to keep all the files together?

It does seem like we need a better place overall to store reference data.

Reply:
I  pulled the references from the existing static json [here](https://github.com/HumanCellAtlas/pipeline-tools/blob/master/adapter_pipelines/Optimus_mouse/adapter_example_static.json), copying any that weren't in public locations to a new public location, but I have no insight into which ones are correct to use.

Reply:
OK, the work to fix up the mouse reference data can be pushed to a different task. It can be fixed up later after the adaptor's in place.